### ISSUE TITLE
Unable to install: non-boolean condition in if statement

### ISSUE THREAD

**Issue by solocommand** [aws-samples/http-requests-mirroring#13]
It looks as if the changes in #4 may have had unintended consequences -- when attempting to boot this up, the cloud init script fails to install the package:
```sh
$ go install github.com/aws-samples/http-requests-mirroring@latest
go: finding module for package github.com/google/gopacket/examples/util
go: finding module for package github.com/google/gopacket
go: finding module for package github.com/google/gopacket/layers
go: finding module for package github.com/google/gopacket/pcap
go: finding module for package github.com/google/gopacket/tcpassembly
go: finding module for package github.com/google/gopacket/tcpassembly/tcpreader
go: found github.com/google/gopacket in github.com/google/gopacket v1.1.19
go: found github.com/google/gopacket/examples/util in github.com/google/gopacket v1.1.19
go: found github.com/google/gopacket/layers in github.com/google/gopacket v1.1.19
go: found github.com/google/gopacket/pcap in github.com/google/gopacket v1.1.19
go: found github.com/google/gopacket/tcpassembly in github.com/google/gopacket v1.1.19
go: found github.com/google/gopacket/tcpassembly/tcpreader in github.com/google/gopacket v1.1.19
# github.com/aws-samples/http-requests-mirroring
go/pkg/mod/github.com/aws-samples/http-requests-mirroring@v0.0.0-20230821134006-28804333813c/main.go:158:5: non-boolean condition in if statement
```

**Comment by mateon01** [comment]
The keepHostHeader variable is of type *bool, which means it's a pointer to a boolean value. As such, it can't be used directly in the condition of an if statement; you have to dereference the contents of the pointer to get the actual boolean value.

You can change code blow
`
if *keepHostHeader {
    forwardReq.Host = req.Host
}
`

**Comment by SimonePomata** [comment]
fixed by #14
